README.md

```diff
@@ -7,15 +7,31 @@
 **Note:** for the command-line interface to `counterparty-lib`, see [`counterparty-cli`](https://github.com/CounterpartyXCP/counterparty-cli).
 
 
-# Installation
+# Getting Started
 
-**WARNING** The `master` branch should only be used as a development target. For production, use only the latest tagged release.
+The simplest way to get your Counterparty node up and running is to use Docker Compose.
+
+```
```

Comment:
Using **```bash**  instead of **```** will add a "play" button in all important IDEs and so the user will be able to just click on it and the command will run.

Example:
```bash
sudo apt install docker-compose
```

Will be displayed like 
<img width="398" alt="Screenshot 2024-01-27 at 13 25 46" src="https://github.com/CounterpartyXCP/counterparty-lib/assets/12685981/3c6be154-86ee-4352-92af-de03a12d8b03">

Reply:
wait how do I do that? that's amazing!

Reply:
@adamkrellenstein you just need to add the text **bash** after the first \```

Example:
\```bash
sudo apt install docker-compose
\```

Reply:
nice!